<launch>
  <node pkg="rr_platform" type="motor_relay_node" name="motor_relay_node" output="screen" respawn="false">
    <param name="speed_topic" type="string" value="/speed" />
    <param name="steering_topic" type="string" value="/steering" />
    <param name="serial_port" type="string" value="/dev/ttyUSB0" />
    <param name="p_const" type="float" value="0.01" />
    <param name="i_const" type="float" value="0.00" />
    <param name="d_const" type="float" value="0.00" />
  </node>
</launch>
